src/Neo/SmartContract/Native/CryptoLib.cs

```diff
@@ -77,6 +77,49 @@ public static byte[] Keccak256(byte[] data)
             return data.Keccak256();
         }
 
+        /// <summary>
+        /// Verifies that a digital signature is appropriate for the provided key and message using the ECDSA algorithm.
+        /// </summary>
+        /// <param name="message">The signed message.</param>
+        /// <param name="signature">The signature to be verified.</param>
+        /// <param name="curveHash">A pair of the curve to be used by the ECDSA algorithm and the hasher function to be used to hash message.</param>
+        /// <returns><see langword="true"/> if the signature is valid; otherwise, <see langword="false"/>.</returns>
+        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 10)]
```

Comment:
```suggestion
        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 10, Name = "ecRecover")]
```
Otherwise this method will have `eCrecover` name in the CryptoLib's manifest.